it/controllers/gains/PolicySummaryControllerISpec.scala

```diff
@@ -157,6 +157,19 @@ class PolicySummaryControllerISpec extends IntegrationTest {
       result.headers("Location").head shouldBe s"/update-and-submit-income-tax-return/additional-information/$taxYear/gains/summary"
     }
 
+    "redirect to the gains summary page when no prior data" in {
+      lazy val result: WSResponse = {
+        clearSession()
+        populateWithSessionDataModel(Seq(completePolicyCyaModel))
+        authoriseAgentOrIndividual(isAgent = false)
```

Comment:
If you add `emptyUserDataStub(nino, taxYear)` here it will go into the None case in the controller as currently it is not.